README.md

```diff
@@ -105,12 +118,19 @@ Tools:
 - [Konga CLI Tool](https://github.com/Floby/konga-cli)
 - [Kong on Tutum](https://github.com/Sillelien/docker-kong)
 - [Kong GUI in JS](https://github.com/rsdevigo/jungle)
-- [Kong GUI in Py](https://github.com/vikingco/django-kong-admin)
+- [Kong Admin in Python](https://github.com/vikingco/django-kong-admin)
 - [Kong UI](https://github.com/msaraf/kong-ui)
 - [Chef Cookbook for Kong](https://github.com/zuazo/kong-cookbook)
+- [Puppet recipie for Kong](https://github.com/scottefein/puppet-nyt-kong)
+- [Ansible role for Kong/Ubuntu](https://github.com/Getsidecar/ansible-role-kong)
 - [Python Client for Kong](https://pypi.python.org/pypi/python-kong/)
 - [Kong with Instaclustr](https://www.instaclustr.com/products/kong/)
 - [.NET Client for Kong](https://www.nuget.org/packages/Kong/0.0.4)
+- [Bonobo: Kong Key Managment (with Mashery migration scripts)](https://github.com/guardian/bonobo)
+- [Adds Monit to base Kong Image](https://github.com/articulate/docker-kong-wait)
```

Comment:
suggesting to be more descriptive as to what the image actually do, aka what it is monitoring for.

"Kong image waiting for Cassandra " works too, is less abstract.